libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -652,6 +656,37 @@ const AP_Param::GroupInfo NavEKF3::var_info2[] = {
     // @Path: ../AP_NavEKF/AP_NavEKF_Source.cpp
     AP_SUBGROUPINFO(sources, "SRC", 1, NavEKF3, AP_NavEKF_Source),
 
+    // @Param: DRAG_BCOEF_X
+    // @DisplayName: Ballistic coefficient measured in X direction
```

Comment:
Ideally the parameter display name should have "drag" or "wind" or something at the front to allow users to understand what it is for without looking at the parameter name.  It's not a big deal but as it stands, the reader will likely wonder "Ballistic coefficient" ... of what?

Reply:
We don't want the display name to imply it is a drag coefficient and also this is an advanced parameter. I will modify them to make 'drag' appear in the display name, but it won't be at the front.